Flaky test - mysqlreceiver - TestMySqlIntegration

Seen here: https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/4716454123?check_suite_focus=true

```
=== CONT  TestMySqlIntegration/Running_mysql_version_8.0
2022/01/05 15:24:53 Starting container id: a6c2212a8acf image: testcontainers/ryuk:0.3.3
2022/01/05 15:24:54 Waiting for container id a6c2212a8acf image: testcontainers/ryuk:0.3.3
2022/01/05 15:24:54 Container is ready id: a6c2212a8acf image: testcontainers/ryuk:0.3.3
2022/01/05 15:25:25 Starting container id: 06cbd6976802 image: 26949dea-4b1c-43e0-9de4-484f831a23de:e6214234-88e7-43a2-b23b-85aa846aa487
2022/01/05 15:25:25 Starting container id: 3620bc846cce image: f10c2a98-16ad-400f-a16e-fd32e76761db:16593a5a-d1ab-43cc-93e8-6cb21ad25938
2022/01/05 15:25:25 Waiting for container id 3620bc846cce image: f10c2a98-16ad-400f-a16e-fd32e76761db:16593a5a-d1ab-43cc-93e8-6cb21ad25938
2022/01/05 15:25:25 Waiting for container id 06cbd6976802 image: 26949dea-4b1c-43e0-9de4-484f831a23de:e6214234-88e7-43a2-b23b-85aa846aa487
2022/01/05 15:25:42 Container is ready id: 06cbd6976802 image: 26949dea-4b1c-43e0-9de4-484f831a23de:e6214234-88e7-43a2-b23b-85aa846aa487
2022/01/05 15:25:43 Container is ready id: 3620bc846cce image: f10c2a98-16ad-400f-a16e-fd32e76761db:16593a5a-d1ab-43cc-93e8-6cb21ad25938
coverage: 93.1% of statements
panic: test timed out after 1m0s

goroutine 503 [running]:
testing.(*M).startAlarm.func1()
	/opt/hostedtoolcache/go/1.17.5/x64/src/testing/testing.go:1788 +0xbb
created by time.goFunc
	/opt/hostedtoolcache/go/1.17.5/x64/src/time/sleep.go:180 +0x4a

goroutine 1 [chan receive]:
testing.(*T).Run(0xc000450680, {0xf0129e, 0x14}, 0xf27490)
	/opt/hostedtoolcache/go/1.17.5/x64/src/testing/testing.go:1307 +0x752
testing.runTests.func1(0x0)
created by github.com/go-sql-driver/mysql.(*mysqlConn).startWatcher
	/home/runner/go/pkg/mod/github.com/go-sql-driver/mysql@v1.6.0/connection.go:611 +0x1b9

goroutine 469 [select]:
net/http.(*persistConn).writeLoop(0xc0000d3440)
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:2386 +0x1a9
created by net/http.(*Transport).dialConn
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:1748 +0x2f8b

goroutine 468 [IO wait]:
internal/poll.runtime_pollWait(0x7fb7523f6c78, 0x72)
	/opt/hostedtoolcache/go/1.17.5/x64/src/runtime/netpoll.go:234 +0x89
internal/poll.(*pollDesc).wait(0xc000503498, 0xc0003b5000, 0x0)
	/opt/hostedtoolcache/go/1.17.5/x64/src/internal/poll/fd_poll_runtime.go:84 +0xbd
internal/poll.(*pollDesc).waitRead(...)
	/opt/hostedtoolcache/go/1.17.5/x64/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Read(0xc000503480, {0xc0003b5000, 0x1000, 0x1000})
	/opt/hostedtoolcache/go/1.17.5/x64/src/internal/poll/fd_unix.go:167 +0x419
net.(*netFD).Read(0xc000503480, {0xc0003b5000, 0x1000, 0x1000})
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/fd_posix.go:56 +0x51
net.(*conn).Read(0xc000010ff8, {0xc0003b5000, 0x1000, 0x1000})
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/net.go:183 +0xb1
net/http.(*persistConn).Read(0xc0000d3440, {0xc0003b5000, 0x1000, 0x1000})
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:1926 +0x110
bufio.(*Reader).fill(0xc0002770e0)
	/opt/hostedtoolcache/go/1.17.5/x64/src/bufio/bufio.go:101 +0x294
bufio.(*Reader).Peek(0xc0002770e0, 0x1)
	/opt/hostedtoolcache/go/1.17.5/x64/src/bufio/bufio.go:139 +0xcc
net/http.(*persistConn).readLoop(0xc0000d3440)
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:2087 +0x2db
created by net/http.(*Transport).dialConn
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:1747 +0x2ef8

goroutine 485 [select]:
go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping.func1()
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.41.1-0.20211222180302-3db1d1146483/receiver/scraperhelper/scrapercontroller.go:171 +0x236
created by go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.41.1-0.20211222180302-3db1d1146483/receiver/scraperhelper/scrapercontroller.go:162 +0x91

goroutine 442 [select]:
net/http.(*persistConn).writeLoop(0xc0000d2ea0)
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:2386 +0x1a9
created by net/http.(*Transport).dialConn
	/opt/hostedtoolcache/go/1.17.5/x64/src/net/http/transport.go:1748 +0x2f8b

goroutine 499 [select]:
github.com/go-sql-driver/mysql.(*mysqlConn).startWatcher.func1()
	/home/runner/go/pkg/mod/github.com/go-sql-driver/mysql@v1.6.0/connection.go:614 +0x112
created by github.com/go-sql-driver/mysql.(*mysqlConn).startWatcher
	/home/runner/go/pkg/mod/github.com/go-sql-driver/mysql@v1.6.0/connection.go:611 +0x1b9

goroutine 484 [select]:
database/sql.(*DB).connectionOpener(0xc000291450, {0x1016320, 0xc000332c80})
	/opt/hostedtoolcache/go/1.17.5/x64/src/database/sql/sql.go:1196 +0xfa
created by database/sql.OpenDB
	/opt/hostedtoolcache/go/1.17.5/x64/src/database/sql/sql.go:794 +0x307

goroutine 447 [select]:
go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping.func1()
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.41.1-0.20211222180302-3db1d1146483/receiver/scraperhelper/scrapercontroller.go:171 +0x236
created by go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.41.1-0.20211222180302-3db1d1146483/receiver/scraperhelper/scrapercontroller.go:162 +0x91
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mysqlreceiver	60.055s
?   	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mysqlreceiver/internal/metadata	[no test files]
FAIL
```

--- a/Makefile.Common
+++ b/Makefile.Common
@@ -15,7 +15,7 @@ ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))) 2>/dev/null)
 # build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
 GO_BUILD_TAGS=""
 GOTEST_OPT?= -race -v -timeout 300s --tags=$(GO_BUILD_TAGS)
-GOTEST_INTEGRATION_OPT?= -race -timeout 60s
+GOTEST_INTEGRATION_OPT?= -race -timeout 120s
 GOTEST_OPT_WITH_COVERAGE = $(GOTEST_OPT) -coverprofile=coverage.txt -covermode=atomic
 GOTEST_OPT_WITH_INTEGRATION=$(GOTEST_INTEGRATION_OPT) -v -tags=integration,$(GO_BUILD_TAGS) -run=Integration -coverprofile=integration-coverage.txt -covermode=atomic
 GOTEST=go test

